### PR TITLE
[@types/react-native-fbsdk] Update typings for 3.0.0

### PR DESCRIPTION
**Pull request by WoonHaKim**
This PR Changes few interface & types of react-native-fbsdk 2.0.0 and 3.0.0
- Add Applink (v3.0.0)
- Add getters for anonymousID, attributionID and advertiserID (v3.0.0)
- Removed the loginBehaviorIOS from <LoginButton /> (v2.0.0)
- Removed ShareApi (v2.0.0)

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/facebook/react-native-fbsdk/releases/tag/v2.0.0>
<https://github.com/facebook/react-native-fbsdk/releases/tag/v3.0.0>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by danger-public** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## react-native-fbsdk ([<kbd>unpkg</kbd>](https://unpkg.com/browse/react-native-fbsdk@latest/))
was missing the following properties:
1. AppLink
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 7068fb6ae249d1d870e65d568ea8eeffc99d7431
</p>

File: types/react-native-fbsdk/index.d.ts [hunk 1]
```diff
@@ -1,18 +1,20 @@
-// Type definitions for react-native-fbsdk 1.1
+// Type definitions for react-native-fbsdk 3.0
 // Project: https://github.com/facebook/react-native-fbsdk
 // Definitions by: Ifiok Jr. <https://github.com/ifiokjr>
 //                 Thibault Malbranche <https://github.com/titozzz>
 //                 Stuart Forrest <https://github.com/stuartforrest-infinity>
+//                 Un Ha Kim <https://github.com/UHKim>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 4.1
```
Comment (andrewbranch) [inline types/react-native-fbsdk/index.d.ts:8]: This should be set as low as possible. Did it need to change? It doesn’t look like it.

**Review by andrewbranch — CHANGES_REQUESTED** [review]

File: types/react-native-fbsdk/index.d.ts (hunk 1 above)
Comment (UHKim) [reply types/react-native-fbsdk/index.d.ts]: Thank you for advice! @andrewbranch

**Review by andrewbranch — APPROVED** [review]
Thanks!

**Comment by Titozzz** [comment]
Ready to merge

**Pull request merged**